OWNERS

```diff
@@ -1,22 +1,11 @@
 # Reviewers can /lgtm /approve but not sufficient for auto-merge without an
 # approver
 reviewers:
-- Rajakavitha1
-- stewart-yu
-- xiangpengzhao
-- zhangxiaoyu-zidif
+- ClaudiaJKang
+- ianychoi
 
 # Approvers have all the ability of reviewers but their /approve makes
 # auto-merge happen if a /lgtm exists, or vice versa, or they can do both
 # No need for approvers to also be listed as reviewers
 approvers:
-- bradamant3
```

Comment:
These approvers and the above reviewers shouldn't be removed, right?

Reply:
Oh, I didn't know. I'll revert the commit.

Reply:
@gochist @xiangpengzhao 👋 We talked about OWNERS members in last night's SIG Docs meeting--yes, it's preferable for a repo to have its own unique set of reviewers/approvers. This is because most of the inherited reviewers from k/website don't speak Korean, for example. 😄 🇰🇷